src/svcutilcore/src/Microsoft/Tools/ServiceModel/SvcUtil/InputModule.cs

```diff
@@ -150,15 +150,6 @@ private bool LoadFileAsAssembly(FileInfo fileInfo)
 
                 _newInputModule.Assemblies.Add(assembly);
 
-                try
-                {
```

Comment:
Why was this code removed? ?This looks like validation code and SetAllowedModes has side effects (it will store the value of the last parameter).

Reply:
I don't think the code is useful anymore. We only have two mode. One is display, which will only be called when you pass help parameter. And the other is XmlSerializer. The call here is to set the default allowed mode is MetadataFromAssembly, and the allowed mode is ToolMode.MetadataFromAssembly , ToolMode.DataContractExport, ToolMode.Validate ,ToolMode.XmlSerializerGeneration. The validate here is to validate the service and not related to xmlserializer.